integration-cli/docker_cli_run_unix_test.go

```diff
@@ -178,26 +178,26 @@ func TestRunContainerWithCgroupParentAbsPath(t *testing.T) {
 func TestRunDeviceDirectory(t *testing.T) {
 	testRequires(t, NativeExecDriver)
 	defer deleteAllContainers()
-	cmd := exec.Command(dockerBinary, "run", "--device", "/dev/snd:/dev/snd", "busybox", "sh", "-c", "ls /dev/snd/")
+	cmd := exec.Command(dockerBinary, "run", "--device", "/dev/:/dev/", "busybox", "sh", "-c", "ls /dev/")
```

Comment:
I think this is testing that a dir of devices are being loaded, which is what is what `/dev/snd` is.